bot/cogs/help_channels.py

```diff
@@ -481,7 +492,6 @@ async def move_to_available(self) -> None:
             f"Ensuring that all channels in `{self.available_category}` have "
```

Comment:
This logging statement can be removed as we're no longer doing the ensure perm thing.

Reply:
Removed.